src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -488,6 +488,10 @@ private function getArgumentsAsPhp(\DOMElement $node, $name, $file, $lowercase =
                     $arguments[$key] = new Reference($arg->getAttribute('id'), $invalidBehavior);
                     break;
                 case 'expression':
+                    if (!class_exists(Expression::class)) {
+                        throw new \LogicException(sprintf('type="expression" cannot be used without the expression component. Try running "composer require symfony/expression-language".'));
```

Comment:
The `symfony/` part is always optional when using Flex. Could we use `composer require expression-language` instead?

Reply:
Since this is inside a component, it feels too presumptive to put the Flex alias.

Reply:
What about:
`The type="expression" attribute cannot be used without the ExpressionLanguage component. Try running "composer require symfony/expression-language".` (we already use that `ExpressionLanguage` name in other similar exceptions.)